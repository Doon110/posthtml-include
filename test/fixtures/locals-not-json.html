<h1>index</h1><include src="./test/fixtures/includes/3.html" locals='{{ response }}'></include>
