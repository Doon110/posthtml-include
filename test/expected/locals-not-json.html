<h1>index</h1><h2>{{ text }}</h2>
